docker-compose.yaml

```diff
@@ -1,32 +1,32 @@
-version: '2'
+version: '3.2'
 
 services:
-  service_php:
+  php:
     build: docker/php
-    tty: true
```

Comment:
IMO, it has no interest for the project.

Reply:
no idea actually, I took that from another symfony project
So I'm sure your right